core/domain/html_cleaner.py

```diff
@@ -262,8 +263,8 @@ def convert_to_text_angular(html_data):
                     link = soup.new_tag('oppia-noninteractive-link')
                     url = tag['href']
                     text = tag.get_text()
-                    link['url-with-value'] = url
-                    link['text-with-value'] = text
+                    link['url-with-value'] = "&quot;" + url + "&quot;"
```

Comment:
(Here and below) Unless you have a good reason to, stick to single quotes here for consistency.

Reply:
Done!